apiextensions/storageversion/migrator.go

```diff
@@ -55,10 +58,14 @@ func NewMigrator(d dynamic.Interface, a apixclient.Interface) *Migrator {
 // Finally the migrator will update the CRD's status and drop older storage
 // versions
 func (m *Migrator) Migrate(ctx context.Context, gr schema.GroupResource) error {
+	logger := logging.FromContext(ctx)
```

Comment:
There's no logger on the context this ~will panic~ return the default logger which is not the same as the one in the `cmd`. Maybe follow the same approach as `IgnoreNotFound` and it can be an optional property

Reply:
Done - thanks for the review with suggestions!!